test-case-reporting/test/testing_utils.ts

```diff
@@ -25,3 +27,109 @@ export async function truncateAll(db: Database): Promise<void> {
   await db('jobs').truncate();
   await db('builds').truncate();
 }
+
+/**
+ * Fills the database with builds, test cases, jobs, and test runs.
+ * Stores 3 test cases.
+ * Stores 2 builds: the first one has 5 jobs and
+ * 15 test runs (one per (job, test case) pair).
+ * The second one has 1 job and 1 test run.
+ * @param db The database we want to fill.
+ */
```

Comment:
this comment will be incorrect if someone changes fillDatabase -- Is it too brittle to use specific numbers in this comment? Or is it fine because it needs to be hardcoded anyways?

Reply:
Should be fine to leave it